pkg/controller/controller.go

```diff
@@ -299,7 +299,10 @@ func (c *CSIControllerService) DeleteVolume(ctx context.Context, req *csi.Delete
 		ll.Warningf("Context canceled or timed out")
 		return nil, status.Error(codes.DeadlineExceeded, "Context canceled or timed out")
 	}
-	c.svc.UpdateCRsAfterVolumeDeletion(ctxWithID, req.VolumeId)
+	if err := c.svc.UpdateCRsAfterVolumeDeletion(ctxWithID, req.VolumeId); err != nil {
+		ll.Errorf("Unable to update CRs after volume deletion: %s", err.Error())
```

Comment:
`ll.Errorf("Unable to update CRs after volume deletion: %s", err)` should be enough.